setup.py

```diff
@@ -701,7 +701,7 @@ def finalize_options(self):
 
     def run(self):
         from distutils.version import StrictVersion
```

Comment:
actually would you mind removing this import, looks like it wont be used anymore.